src/components/TechnicalInfoContent/TechnicalInfoContent.js

```diff
@@ -0,0 +1,105 @@
+// Copyright (c) Cosmo Tech.
+// Licensed under the MIT license.
+
+import React from 'react';
+import { Button, Grid, List, ListItem, Typography } from '@mui/material';
+import ConfigService from '../../services/ConfigService';
+import { useSolutionData } from '../../state/hooks/SolutionHooks';
+import { useWorkspaceData } from '../../state/hooks/WorkspaceHooks';
+import { useTranslation } from 'react-i18next';
+
+export const TechnicalInfoContent = () => {
+  const { t } = useTranslation();
+  const solutionData = useSolutionData();
+  const solutionName = solutionData?.name ?? '';
+  const solutionDescription = solutionData?.description ?? '';
+  const currentWorkspace = useWorkspaceData();
+
+  const technicalInformation = [
+    {
+      id: 'webappVersion',
+      label: t('genericcomponent.dialog.technicalInfo.webAppVersion', 'Webapp version'),
+      content: ConfigService.getParameterValue('APP_VERSION'),
+    },
+    {
+      id: 'registryName',
+      label: t('genericcomponent.dialog.technicalInfo.repositoryName', 'Container registry repository name'),
+      content: solutionData?.repository,
+    },
+    {
+      id: 'registryTag',
+      label: t('genericcomponent.dialog.technicalInfo.repositoryTag', 'Container registry repository tag'),
+      content: solutionData?.version,
+    },
+    {
+      id: 'SDKVersion',
+      label: t('genericcomponent.dialog.technicalInfo.sdkVersion', 'SDK version'),
+      content: solutionData?.sdkVersion,
+    },
+  ];
+
+  const infoToDisplay = currentWorkspace
+    ? technicalInformation
+    : technicalInformation.filter((infoItem) => ['webappVersion'].includes(infoItem.id));
+
+  const copyInfoToClipboard = () => {
+    navigator.clipboard.writeText(
+      solutionName +
+        '\n' +
+        solutionDescription +
+        '\n' +
+        infoToDisplay
+          .map(
+            (infoItem) =>
+              infoItem.label +
+              ' ' +
+              (infoItem.content ??
+                t('genericcomponent.dialog.technicalInfo.notSpecifiedEntry') + ' (' + infoItem.content + ')')
+          )
+          .join('\n')
+    );
+  };
```

Comment:
This function could probably be rewritten with less lines and be easier to read :slightly_smiling_face: 

Something like this:
```const copyInfoToClipboard = () => {
  const writeInfoItem = (infoItem) => {
    const itemContent =
      infoItem.content ?? t('genericcomponent.dialog.technicalInfo.notSpecifiedEntry') + ' (' + infoItem.content + ')';
    return `${infoItem.label} ${itemContent}`;
  };

  navigator.clipboard.writeText(
    solutionName + '\n' + solutionDescription + '\n' + infoToDisplay.map(writeInfoItem).join('\n')
  );
};
```